introspection_response_writer.go

```diff
@@ -202,6 +202,11 @@ func (f *Fosite) WriteIntrospectionResponse(rw http.ResponseWriter, r Introspect
 		return
 	}
 
+	expiresAt := int64(0)
```

Comment:
Can you add a test (or modify an existing one) for this?

Reply:
Sure, will do later.